src/App.tsx

```diff
@@ -20,12 +27,16 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(e) => {
+                  setQuery(e.target.value);
+                }}
```

Comment:
_comment_
In several works, mentors mentioned to avoid passing functions inline. It is advised to create a funtion to handle this change and change state inside it.